examples/docs/src/components/sidebar.js

```diff
@@ -0,0 +1,203 @@
+import React from "react";
+import { StaticQuery, graphql } from "gatsby";
+import styled from "react-emotion";
+import flatten from "lodash.flatten";
+import { ExternalLink } from "react-feather";
+import Link from "./link";
+
+// prettier-ignore
+const nav = [
```

Comment:
this whole thing should be hard coded as JSX or programmatically generated. This "JSON Spec" approach is an anti-pattern for non-data content (that is, data that would display in a large table or infinite list, etc).

I'm mostly concerned about having to maintain a manual list of all of the documentation. Adding a file should "just work" when it comes to being in the navigation, displayed on the site, etc. Other than the top level nav, use alphabetical sorting.